modules/basegl/src/processors/imageprocessing/imagechannelselect.cpp

```diff
@@ -50,7 +50,7 @@ ImageChannelSelect::ImageChannelSelect()
     addProperty(channelSelector_);
 }
 
-void ImageChannelSelect::preProcess(TextureUnitContainer &cont) {
+void ImageChannelSelect::preProcess(TextureUnitContainer &) {
```

Comment:
Wonder why I didn't get this warning, I made a clean build of the module to fix warnings.